project_sla/security/ir.model.access.csv

```diff
@@ -6,3 +6,4 @@ access_sla_lines_user,access_sla_lines_user,model_project_sla_line,base.group_us
 access_sla_control_manager,access_sla_control_manager,model_project_sla_control,project.group_project_manager,1,1,0,0
 access_sla_control_user,access_sla_control_user,model_project_sla_control,base.group_user,1,0,0,0
 access_sla_controlled_manager,access_sla_controlled_manager,model_project_sla_controlled,project.group_project_manager,1,1,1,1

```

Comment:
I guess these are not necessary, since they are already defined in `project_sla`.